airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py

```diff
@@ -254,6 +254,7 @@ def get_custom_insights_streams(self, api: API, config: ConnectorConfig) -> List
                 )
                 raise AirbyteTracedException(
                     message=message,
+                    internal_message=message,
```

Comment:
Are we reading only `internal_message` field for reporting DB purposes ?

Reply:
When the exception is logged it uses the `internal_message` field. Previously it was None and we weren't getting the message. Now the framework will log the message, and rudder sources will get all the error messages.